src/client/settings/_settings.coffee

```diff
@@ -26,10 +26,17 @@ angular.module 'app.settings', []
   vm.load = ->
     $http.get('/containers/settings/settings').success (data) ->
       vm.settings = data
- 
+
+  vm.start_screens = ["Movies", "TV Series", "Anime"]
```

Comment:
for now we can let like that but in the future it should based on plugin available i guess.